### ISSUE TITLE
Lox model shading/disappearing on far zoom out

### ISSUE THREAD

**Issue by apgeo** [aardgoose/CaveView.js#8]
On lox models, if you zoom out enough, at some point, the model disappears and looks to be "shaded" by a black region. My guess is that that black covering layer is the bounding box of the 3d world in which the model is rendered. You can try this with "cheddar.lox" example file.

While on "cheddar.lox" and others that might not be a big problem since normally you don't need to zoom out so much, I have another sample file which has a different geometry and gets shaded as soon as you zoom out and can't see the entire model. The cave model is smaller in comparison with the surface model part which makes this being rendered differently - you have to zoom out much more to see the entire terrain. I can provide that example.

**Comment by aardgoose** [comment]
I think you are seeing the clipping box of the 'camera', I've kept this conservatively small to best utilise the GPU depth buffer, The majority of devices appear to have 24bit depth buffers, which works fine with a larger clipping box.  I've increased the clipping box size by 8 times which eliminates clipping at practical zoom levels.

I've also moved the camera 'further back' for orthographic viewing which stops apparent clipping of near objects - actually they are behind the camera location.

I'll push out a new version soon.

https://github.com/aardgoose/CaveView.js/commit/245987c11a8124231e5b25ab50d7d13b22e8f670
